WebUI/electron/main.ts

```diff
@@ -537,6 +537,17 @@ function isProcessRunning(pid: number) {
 function wakeupApiService() {
   const wordkDir = path.resolve(app.isPackaged ? path.join(process.resourcesPath, "service") : path.join(__dirname, "../../../service"));
   const baseDir = app.isPackaged ? process.resourcesPath : path.join(__dirname, "../../../");
+  
+  try {
+    // replace `torchvision.transforms.functional_tensor` with `torchvision.transforms.functional` in `degradations.py`
+    const basicSrPathFileToBePatched = path.resolve(path.join(baseDir, "env/Lib/site-packages/basicsr/data/degradations.py"));
+    const fileContent = fs.readFileSync(basicSrPathFileToBePatched, 'utf8');
+    const patchedContent = fileContent.replace('torchvision.transforms.functional_tensor', 'torchvision.transforms.functional');
+    fs.writeFileSync(basicSrPathFileToBePatched, patchedContent, 'utf8');
+  } catch (error) {
+    logger.error(`Could not patch torchvision.transforms.functional_tensor in degradations.py: ${error}`);
+  }
+
```

Comment:
I've just fixed this issue in https://github.com/intel/AI-Playground/commit/9cb36b407247acecccfdb1939671ed0e52624a1f (probably in a more portable way).
You can safely revert this change.

Reply:
Nice! Thank you!